x-pack/plugin/vectors/src/main/java/org/elasticsearch/xpack/vectors/query/ScoreScriptUtils.java

```diff
@@ -67,18 +70,10 @@ BytesRef getEncodedVector() {
             } catch (IOException e) {
                 throw ExceptionsHelper.convertToElastic(e);
             }
-
-            // Validate the encoded vector's length.
             BytesRef vector = docValues.getEncodedValue();
             if (vector == null) {
                 throw new IllegalArgumentException("A document doesn't have a value for a vector field!");
             }
-
-            int vectorLength = VectorEncoderDecoder.denseVectorLength(scoreScript._getIndexVersion(), vector);
```

Comment:
It's great we were able to remove these references to `ScoreScript#_getIndexVersion`. I think this method can now be deleted?

Reply:
Great point! Addressed in a66f75590ba961439ac02907a46074312d3a80a2